web/themes/custom/fmcg_material/sass/_button.scss

```diff
@@ -30,3 +30,66 @@ input.button, .btn, a.btn, span.btn, .tabledrag-toggle-weight, input[type="submi
     background: $fmcg-black;
   }
 }
+
+.button-action {
+  background-color: #50C878;
+  color: #FFFFFF;
+
+  &:hover {
+    background-color: #FFFFFF;
+    color: #50C878;
+    border-color: #50C878;
+    transition: none;
+  }
+}
+
+.ui-widget-content a, .change-role a {
+  padding: 8px 12px;
+  height: auto;
+
+  &:hover {
+    padding: 6px 10px;
+    border: 2px solid $fmcg-yellow;
+    color: $fmcg-yellow;
+    background: $fmcg-black;
+  }
+}
+
+.js .dropbutton-widget .dropbutton {
+  overflow: hidden;
+  margin: -12px;
+  padding: 0;
+  list-style-type: none;
+  list-style-image: none;
+}
+
+li.approve, li.reject {
+  background-color: $fmcg-chrome;
+  border-top-left-radius: 2px;
+  border-bottom-left-radius: 2px;
+  padding: 8px 12px;
+
+  a {
+    color: $fmcg-black;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 600;
+    letter-spacing: 0.5px;
+  }
+}
+
+li.approve {
+  margin: 8px;
+}
+
+li.reject {
+  margin: -8px 0px 0px 8px;
+}
```

Comment:
Work out how to to do these without using approve/reject. It might be about :first-child and :last-child?